From 46b000c70f8b5caefe77c7e4b9c3b7ef07044da1 Mon Sep 17 00:00:00 2001
From: Jonathan Paugh <jpaugh@pobox.com>
Date: Sun, 11 Apr 2021 03:58:00 -0500
Subject: [PATCH] Allow xsltproc to download files

These files are required to build the man pages; unfortunately, xsltproc
does not appear to cache these files, so this may not be the best
solution.
---
 man/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/man/meson.build b/man/meson.build
index 2345ea1f..f14b3e32 100644
--- man/meson.build
+++ man/meson.build
@@ -4,7 +4,6 @@ assert(xsltproc.found(), 'xsltproc is required for man pages generation')
 xsltproc_cmd = [
   xsltproc,
   '--output', '@OUTPUT@',
-  '--nonet',
   '--stringparam', 'man.output.quietly', '1',
   '--stringparam', 'funcsynopsis.style', 'ansi',
   '--stringparam', 'man.th.extra1.suppress', '1',
-- 
GitLab

